### DISCUSSION TITLE
Plan for archiving older versions of docs

### DISCUSSION THREAD

**Discussion by g1eny0ung** [chaos-mesh/chaos-mesh#3569] [General]
We still have all versions of the docs on our website:

<img width="220" alt="image" src="https://user-images.githubusercontent.com/15034155/185792686-58934b07-e9d5-4298-8a6b-6b8a51f40f59.png">

But refer to our release cycle, many old versions currently no longer maintained.

I suggest we archive some of our old docs and put them on Netlify.

What do you guys think? @chaos-mesh/maintainers @chaos-mesh/committers Feel free to comment below.

**Comment by YangKeao** [comment]
Yes! These docs can be misleading, if the users enters them through searching in the website. There have been several issues caused by users searching around the website and read documents in different version (but didn't realize).

But there is also another problem: if we simply removed these docs, but the index still exist in general search engine (e.g. google), the user will get a lot of 404 pages while searching the chaos mesh.

**Reply by g1eny0ung** [reply]
> But there is also another problem: if we simply removed these docs, but the index still exist in general search engine (e.g. google), the user will get a lot of 404 pages while searching the chaos mesh.

@YangKeao This is what I planned:

1. I will add some redirects to the old docs. When users access old docs, they will jump to our docs of the current last release version.
2. I will add a new `/versions` page to tell people where our unmaintained docs are located. We can refer to https://docusaurus.io/versions to build it.

    Also, refer to the following screenshot to modify our docs entry:

    <img width="240" alt="image" src="https://user-images.githubusercontent.com/15034155/185860629-b8528a6a-3b69-4407-b32e-c7523eb6e520.png">

We can't let search engines like Google suit our changes rapidly. It's uncontrollable. We can only do things like the above and then wait for the search engine to re-crawl our updates (for example, all redirects will use `<meta />` tags to tell Google to update the search results).

Ref: https://developers.google.com/search/docs/advanced/crawling/301-redirects.

**Comment by g1eny0ung** [comment]
Done. https://chaos-mesh.org/versions/

**Reply by STRRL** [reply]
Cool!

**Comment by dcalvin** [comment]
Good stuff!
